ui/app/components/sidebar/nav/cluster.hbs

```diff
@@ -125,5 +125,10 @@
       @text="Custom Messages"
       data-test-sidebar-nav-link="Custom Messages"
     />
+    <Nav.Link
```

Comment:
I think there's a ticket, but it might be worth include a "todo" for wrapping this in permissions later